simularium/simularium-viewer
# Feature/dev build

Problem
=======
#221 
I previously tried using npm tags to get a build on merge, but the tags are not sub versions as I thought, so you can't publish them without also versioning the app. 

Solution
========
This builds the app, copies everything needed into one folder, and then pushes that to the `build` branch. I've tested this by turning off the conditional that only builds on main, so if you look in the recent actions you can see how it works. 

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)

Steps to Verify:
----------------
1. [action that ran](https://github.com/simularium/simularium-viewer/actions/runs/3595831276)
2. check the build artifact
3. check branch `build`
